sample-app/src/main/java/com/joinforage/android/example/network/ForageApi.kt

```diff
@@ -10,6 +10,6 @@ import retrofit2.http.POST
 
 interface ForageApi {
     @POST("api/payments/")
-    @Headers("API-VERSION: 2023-03-31")
+    @Headers("API-VERSION: 2023-05-15")
```

Comment:
cc comment about separating api-version header to a diff pr

Reply:
Just changed back, added those to test.